tests/golden_tests.rs

```diff
@@ -517,7 +517,7 @@ fn compile_long() {
       ..Default::default()
     };
 
-    compile_book(&mut book, opts.clone(), diagnostics_cfg, None)?;
-    Ok("Compiled".to_string())
+    let res = compile_book(&mut book, opts.clone(), diagnostics_cfg, None)?;
+    Ok(format!("{}{}", res.diagnostics, hvm_book_show_pretty(&res.hvm_book)))
```

Comment:
We don't want to print the book in case of success. This test in specific is made to generate way too large structures.

Since the compile fails it should already be saving the error messages. I think it should work if  you just leave it like it was.